Accessories/Kaman_IO_ConverterBoard/REV20230213C/README.md

```diff
@@ -1,29 +1,37 @@
 # Kaman I/O Converter Board
 
-This folder contains the compiled design outputs for the Kaman I/O Converter REV B. This includes the schematics, images of the PCB layout, and the bill of materials (BOM).
-
-## PCB Properties
-| Question          | Answer        |
-|-------------------|---------------|
-| Size              | 2.4" x 2.5"   |
-| Number of Layers  | 2               |
-| Thickness         | 1.6mm            |
-| Min Track/Spacing | 6/6mil          |
-| Min Hole Size     | 0.3mm           |
-| Surface Finish    | HASL (Tin Lead) |
-| Silkscreen        | Both sides      |
-| Unique Part Count | 15            |
-| SMT Part Count    | 69            |
-| Trough-Hole Count | 3            |
-
-## Recommended Cables
-
-The Kaman I/O Converter board requires cables to interface with the AMDC GPIO connectors. The following are the recommended cables to buy.
-
-### GPIO Cables
-
-<img src="https://media.digikey.com/Photos/Amphenol%20Photos/MFG_CS-DSDHD15MM0.jpg" width="25%" />
-
-Digi-Key Part Number: `CS-DSDHD15MM0-007.5-ND`
-
-Only one cable is required.
+This folder contains the compiled design outputs for the Kaman I/O Converter REV B. This includes the schematics, images of the PCB layout, the bill of materials (BOM), and the files needed to order boards from an online vendor.
+
+
+## Board Parameters
+
+| Question                | Answer             |
+|-------------------------|--------------------|
+| Board Size              | 2.4" x 2.5"        |
+| Numer of Layers         | 2 Layers           |
+| Double Sided            | Yes                |
+| Surface Finish          | Tin Lead           |
+| Silkscreen              | White              |
+| Material                | FR-4, TG150        |
+| Board Thickness         | 0.41 mm (16.2 mil) |
```

Comment:
From what I can tell from looking online, 1.6mm is the standard thickness for 2, 4, and maybe even 6 layer PCBs.

See 
- [Rush River's Guide to PCB Thickness](https://rushpcb.com/guide-to-pcb-thickness/)
- [Altium's Standard PCB Thickness and Your Layer Stack article](https://resources.altium.com/p/standard-board-thickness-and-your-layer-stack)